Unix/buildtool

```diff
@@ -1,5 +1,48 @@
 #!/bin/sh
 
+#
+# oslater <arg1> <arg2>
+# returns true (0) if the os version <arg1> is greater than the os version <arg2>. 
+# The strings are required to be in the form num.num  
+#
+# usage:   if oslater $thisos "16.10" 
+#             then
+#                do_something
+#             else 
+#                complain
+#          fi
+
+oslater ()
+{
+    os1=$1
+    os2=$2
+    
+    osmaj1=${os1%.*}
+    osmin1=${os1#*.}
+    
+    osmaj2=${os2%.*}
+    osmin2=${os2#*.}
+    
+    if [ "$osmaj1" -lt "$osmaj2" ]
+    then 
```

Comment:
Fix formatting to be consistent with rest of file (see line 766 as an example).

Will wait for new iteration since you stated this isn't working properly.